src/query/server/server.go

```diff
@@ -72,6 +72,11 @@ var (
 		Namespace: "default",
 		Retention: 2 * 24 * time.Hour,
 	}
+
+	defaultLimitsConfiguration = &config.LimitsConfiguration{
+		// from prom: This is sufficient for 60s resolution for a week or 1h resolution for a year.
+		MaxComputedDatapoints: 11000,
```

Comment:
Hey since we have a lot of users using our default retention of 4 weeks (720h/30d) which is on a lot of our guides, and also a similar retention that we run with internally, can you perhaps make that fit that case?  

i.e. 1m at 30 days, so 43,200 max computed datapoints?

Reply:
Absolutely, good call!

Reply:
Perhaps we should make it larger than that (86400--double)? Since it's such a crude limit, I'd rather folks not hit it unless they're doing something really awful (would be an annoying UX and probably would not be what saved us from anything)

Reply:
Set it at 86400 for the moment (which also has the nice property of being == 1s for 1 day); lmk if we should do a different number!

Reply:
After a bit more discussion, I actually disabled this by default, so as to not disrupt existing expensive queries. We can flag it on in our internal deployment soon (will keep an eye on 400's).